fusions.wdl

```diff
@@ -90,13 +85,12 @@ task fusions {
         memory: "4GB"
         cpu: cores
         preemptible: 1
-        docker: "chrisamiller/genomic-analysis:0.2"
+        docker: "jonathanztangwustl/docker_fusions:0.1.2"
```

Comment:
Let me know when you have this docker image's code posted somewhere (on github is fine), and i'll clone it, build it, and push it to mgibio

Reply:
Swapped to `mgibio/chip_fusions:0.1.2`